src/components/graph/search/GraphSearchInput.tsx

```diff
@@ -122,14 +122,16 @@ function GraphSearchInput(props: Props) {
 		if (windowRange) {
 			const [from, to] = windowRange;
 			const centerTimestamp = from + (to - from) / 2;
-
-			setInputConfig({
-				isValidDate: true,
-				mask: DATE_TIME_MASK,
-				placeholder: DATE_TIME_PLACEHOLDER,
-				timestamp: centerTimestamp,
-				value: moment.utc(centerTimestamp).format(DATE_TIME_MASK),
-			});
+			if (!inputConfig.timestamp || Math.abs(centerTimestamp - inputConfig.timestamp) > 30000) {
```

Comment:
If you squeeze window then graph will be have strange behavior. I suppose it is the reason

Reply:
I suppose we need big changes in architecture to fix this bug.